framework/PostProcessors/DataMining.py

```diff
@@ -197,6 +198,7 @@ def __init__(self, messageHandler):
 
     self.PreProcessor = None
     self.metric = None
+    self.pivotParameter = None                          ## default pivotParameter for HistorySet
```

Comment:
can you comment these variables above the variables themselves?

Reply:
done